hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -67,14 +67,15 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster, final
                 nodeHosts.add("127.0.0.1:" + (FIRST_GRPC_PORT + (nodeId * 2)) + ":" + account);
             }
             sb.append("\nnextNodeId, ").append(numNodes).append("\n");
-            final var configText = sb.toString();
+            final String configText = sb.toString();
 
             for (int nodeId = 0; nodeId < numNodes; nodeId++) {
-                final var workingDir =
+                final Path workingDir =
                         Path.of("./build/hapi-test/node" + nodeId).normalize();
                 setupWorkingDirectory(workingDir, configText);
-                final var nodeName = NODE_NAMES[nodeId];
-                final var acct = AccountID.newBuilder().accountNum(3L + nodeId).build();
+                final String nodeName = NODE_NAMES[nodeId];
+                final AccountID acct =
+                        AccountID.newBuilder().accountNum(3L + nodeId).build();
```

Comment:
Not a fan of `var` here I guess? :-)

Reply:
Not for primitives and common classes that aren't much longer than `var`.  It just makes the code harder to read in that case as the variable type is unclear.